src/components/helpers/validators/required-fields-validator.test.js

```diff
@@ -0,0 +1,16 @@
+import fieldIsRequired from "./required-fields-validator";
+
+describe("test required fields validator", () => {
```

Comment:
Please give names for `tests` and for `describe` properly according to some conventions like [here](https://markus.oberlehner.net/blog/naming-your-unit-tests-it-should-vs-given-when-then/)

Reply:
Done